concourse/scripts/rds_instances.sh

```diff
@@ -42,6 +42,12 @@ fi
 
 # Make a list of RDS Instances that are assigned to the specific VPC ID, are not
 # MultiAZ and are/do not contain any replicas.
+all_instance_ids=$(
```

Comment:
since `instance_ids` and `instance_ids_after` are generated from the same query/command, maybe it would make sense to make this into a function so that's clear it's the same query for people editing it later.

Reply:
I added it